doc/source/contributor/environments/ci-aio.rst

```diff
@@ -11,8 +11,29 @@ Prerequisites
 * a CentOS Stream 8 or Ubuntu Focal 20.04 host
 * access to the Test Pulp server on SMS lab
 
-Setup
-=====
+Automated Setup
+===============
+
+Access the host via SSH.
+
+Clone this kayobe configuration repository:
+
+.. parsed-literal::
+
+   git clone https://github.com/stackhpc/stackhpc-kayobe-config
+
+Run the setup script:
+
+.. parsed-literal::
+
+   ./stackhpc-kayobe-config/environments/ci-aio/automated-setup.sh
```

Comment:
Wondering if we should instead `curl` or `wget` the script to avoid having two copies of the config repo, which could be confusing?